openlibrary/plugins/importapi/import_validator.py

```diff
@@ -1,24 +1,182 @@
-from typing import Annotated, Any, TypeVar
+# mostly generated by datamodel-codegen:
+#   filename:  import.schema.json
+#   timestamp: 2023-07-24T12:05:23+00:00
 
+from __future__ import annotations
+
+from enum import Enum
+from typing import TypeVar, Optional, Any
 from annotated_types import MinLen
-from pydantic import BaseModel, ValidationError
+from pydantic import (
+    ConfigDict,
+    BaseModel,
+    Field,
+    RootModel,
+    ValidationError,
+    AnyHttpUrl,
+    StringConstraints,
```

Comment:
Keeping imports sorted makes it easy to spot missing imports and nearly impossible to insert duplicates.
```suggestion
    AnyHttpUrl,
    BaseModel,
    ConfigDict,
    Field,
    RootModel,
    StringConstraints,
    ValidationError,
```